src/components/toolbar-search/toolbar-search.component.less

```diff
@@ -30,7 +30,7 @@ ux-toolbar-search {
         }
 
         [uxToolbarSearchField] {
-            padding-right: 35px;
+            padding-right: 20px;
```

Comment:
The text now overlaps the icon so please revert this change and find an alternative solution.
![image](https://user-images.githubusercontent.com/26114101/90615848-8d80da80-e204-11ea-859e-5704f7b73695.png)